packages/remix-cloudflare-workers-esm/worker.ts

```diff
@@ -0,0 +1,174 @@
+import type { Options as KvAssetHandlerOptions } from "@cloudflare/kv-asset-handler";
+import {
+  getAssetFromKV,
+  MethodNotAllowedError,
+  NotFoundError,
+} from "@cloudflare/kv-asset-handler";
+import type { AppLoadContext, ServerBuild } from "@remix-run/cloudflare";
+import { createRequestHandler as createRemixRequestHandler } from "@remix-run/cloudflare";
+import moduleJSON from "__STATIC_CONTENT_MANIFEST";
+
+type AssetManifest = Omit<KvAssetHandlerOptions["ASSET_MANIFEST"], string>;
+type KvAssetHandlerEvent = Parameters<typeof getAssetFromKV>[0];
+type WaitUntilCallback = KvAssetHandlerEvent["waitUntil"];
+
+const module = JSON.parse(moduleJSON) as AssetManifest;
+
+/**
+ * A function that returns the value to use as `context` in route `loader` and
+ * `action` functions.
+ *
+ * You can think of this as an escape hatch that allows you to pass
+ * environment/platform-specific values through to your loader/action.
+ */
+export type GetLoadContextFunction<Env = unknown> = (
+  request: Request,
+  env: Env,
+  ctx: ExecutionContext
+) => AppLoadContext;
+
+export type RequestHandler = ReturnType<typeof createRequestHandler>;
+
+/**
+ * Returns a request handler for the Cloudflare runtime that serves the
+ * Remix SSR response.
+ */
+export function createRequestHandler<Env = unknown>({
+  build,
+  getLoadContext,
+  mode,
+}: {
+  build: ServerBuild;
+  getLoadContext?: GetLoadContextFunction<Env>;
+  mode?: string;
+}) {
+  let handleRequest = createRemixRequestHandler(build, mode);
+
+  return (request: Request, env: Env, ctx: ExecutionContext) => {
+    let loadContext = getLoadContext?.(request, env, ctx);
+
+    return handleRequest(request, loadContext);
+  };
```

Comment:
are the `let`s correct? 🤔 
```suggestion
  const handleRequest = createRemixRequestHandler(build, mode);

  return (request: Request, env: Env, ctx: ExecutionContext) => {
    const loadContext = getLoadContext?.(request, env, ctx);

    return handleRequest(request, loadContext);
  };
```

Reply:
Yeah, this is what their ESLint config prefers. Not sure about the reasoning but I got autocorrected into it.

Reply:
ah.... weird :sweat:

Reply:
Remix code base generally use `let` instead of `const` : https://dev.to/ryanflorence/comment/1k697

Reply:
ah... I see :open_mouth: 

Thanks for the link, very informative :slightly_smiling_face: